package.json

```diff
@@ -212,7 +212,7 @@
     "version": "oclif-dev readme && git add README.md",
     "lint": "eslint src/**/*.ts deployer/src/**/*.ts --ignore-pattern node_modules",
     "minify": "test $NO_MINIFY || for i in `find lib -type f -name \\*.js`; do npx minify $i > minified; mv minified $i; done",
-    "version-check": "npx check-node-version --node \">= 10.0\""
```

Comment:
I'm taking your word on this change but could you explain the context better?  We have a similar check in a different repo also.  There we mandate node 14.6.   I don't recall any problems.

Reply:
I had the check fail on my environment although it wasn't clear to me immediately what the issue was. I had to test numerous configs before I got it working. I'm not wondering whether this could be at least an issue due to me using zsh rather than bash? I did test the command with numerous different node versions to confirm it worked as expected (although I feel a little unsure as to why it failed initially).

Reply:
It does look like a shell meta-character quoting issue.   I use bash and with this change a backslash is retained in the final string and confuses `check-node-version`.   As you might have guessed, it worked for me without the change.

I think this is a case where the need to package the check in an `npm` install conflicts with the ability to do it cleanly (without being shell-dependent).   The plan is

- I will revert the change temporarily since it breaks bash users (I believe)
- I will open a separate issue to address doing this check is a shell-independent fashion